### PR TITLE
fix: create cornerstone learner audit records with new foreign keys

### PR DESCRIPTION
**Pull request by johnnagro**
## Description

Recent bugs have pointed out that we were creating `CornerstoneLearnerDataTransmissionAudit` records without the newly required `enerprise_customer_uuid` nor `plugin_configuration_id` foreign keys. We use these keys to differentiate records when a customer has more than 1 configuration of the same kind. Cornerstone creates these records in a web view as learners click through - we had missed updating that area. This corrects/amends that. Includes a management back-fill command to update stuff we've already orphaned.

I expanded the testing of the Cornerstone click-through logic. I expanded the logging in the click-through logic. I formatted all the new and existing logging to use the standard splunk key/value pairs we rely on for debugging.

## References

- https://2u-internal.atlassian.net/browse/ENT-6033
- https://2u-internal.atlassian.net/browse/ENT-6051

### CODE REVIEW

File: integrated_channels/integrated_channel/management/commands/backfill_missing_csod_foreign_keys.py
Comment (alex-sheehan-edx) [inline integrated_channels/integrated_channel/management/commands/backfill_missing_csod_foreign_keys.py:1]: you're going to run this through jenkins correct?

File: tests/test_enterprise/views/test_router_view.py
Comment (alex-sheehan-edx) [inline tests/test_enterprise/views/test_router_view.py:228]: since this is required to be contained by the base url in the test config defined in the setup, would it be smart to keep it as a constant, perhaps as a class var? then you can construct the base url with said constant

File: tests/test_management.py
Comment (alex-sheehan-edx) [inline tests/test_management.py:1787]: I thought this was taken care of for you with the fixtures 🤔

**Review by alex-sheehan-edx — COMMENTED** [review]
looks pretty good, Waiting for small change before approval

File: integrated_channels/integrated_channel/management/commands/backfill_missing_csod_foreign_keys.py
Comment (johnnagro) [reply integrated_channels/integrated_channel/management/commands/backfill_missing_csod_foreign_keys.py:1]: correct

File: tests/test_management.py
Comment (johnnagro) [reply tests/test_management.py:1787]: i saw the same pattern above, so i followed suit

File: integrated_channels/integrated_channel/management/commands/backfill_missing_csod_foreign_keys.py
Comment (alex-sheehan-edx) [reply integrated_channels/integrated_channel/management/commands/backfill_missing_csod_foreign_keys.py:1]: Cool! Will look for the internal PR

**Review by alex-sheehan-edx — APPROVED** [review]
lgtm!

**Pull request merged**
